parts/linux/cloud-init/artifacts/cse_install.sh

```diff
@@ -16,7 +16,7 @@ SECURE_TLS_BOOTSTRAP_KUBELET_EXEC_PLUGIN_DOWNLOAD_DIR="/opt/azure/tlsbootstrap"
 SECURE_TLS_BOOTSTRAP_KUBELET_EXEC_PLUGIN_VERSION="v0.1.0-alpha.2"
 TELEPORTD_PLUGIN_DOWNLOAD_DIR="/opt/teleportd/downloads"
 TELEPORTD_PLUGIN_BIN_DIR="/usr/local/bin"
-CONTAINERD_WASM_VERSIONS="v0.3.0 v0.5.1 v0.8.0"
+CONTAINERD_WASM_VERSIONS="v0.8.0"
```

Comment:
Could we have a follow-up PR to upgrade the shim to the [latest version](https://github.com/deislabs/containerd-wasm-shims/releases/tag/v0.11.1)?

Reply:
Do you want v0-8-0 switched to v0-11-1? Or do you want us to bump all versions up like below:

v0-3-0 ==> vX-X-X
v0-5-1 ==> vX-X-X
v0-8-0 ==> vX-X-X

If so, which versions would map to the older ones? 

We are just looking to get the failing curl command to wws for v0-3-0 and v0-5-1 out of the code, as its adding 5 minutes to our build time due to retries.

Reply:
Thank you for the review